src/js/containers/aboutTheData/AgencyTableMapping.jsx

```diff
@@ -80,29 +80,35 @@ export const agenciesTableColumns = {
         },
         {
             title: "recent_publication_date",
-            displayName: "Most Recent Publication Date",
+            displayName: "Most Recent Update",
             icon: (<Tooltip title="Most Recent Update" />)
         },
         {
             title: "missing_tas_accounts_count",
-            displayName: "Count of Agency TAS in GTAS Not in File A",
+            displayName: "Number of TAS Missing from Account Balance Data",
             icon: (
-                <Tooltip title="Number of TASs Missing in Account Balance Data" />
+                <Tooltip title="Number of TASs Missing from Account Balance Data" />
             )
```

Comment:
```suggestion
            ),
            right: true
```
Some of these columns are right-aligned in the mockup

Reply:
ah thank you